Add Makefile and update documentation (#149)
aws-samples/cfn101-workshop: README.md

<h1 align="center">
AWS CloudFormation - Workshop
<br>
    <a href="https://cfn101.workshop.aws"><img alt="Website" src="https://img.shields.io/website?down_color=red&down_message=down&up_color=green&up_message=up&url=https%3A%2F%2Fcfn101.workshop.aws"></a>
    <a href="https://github.com/aws-samples/cfn101-workshop/actions"><img alt="GitHub Workflow Status" src="https://github.com/aws-samples/cfn101-workshop/workflows/Unit%20Tests/badge.svg"></a>
</h1>

This repository provides all the resources referenced in the [CloudFormation](https://cfn101.workshop.aws/) workshop as
well as the code used to build it.

## Usage
1. Clone the repository to your working directory or Download the ZIP file from GitHub.
2. Open the downloaded files in your code editor or IDE of your choice.

> The working directory is located in CloudFormation templates and can be found in [code/workspace](code/workspace)

Reviewer: It's not clear to me what this line is saying. What do you mean by 
> The working directory is located in CloudFormation templates

Author: Done

where you can follow along and write your code to.

> In the [code/solutions](code/solutions), you can find final template for each lab. This can be used as a reference,
> in case you get stuck and things don't work the way they should.

Reviewer 2: ```suggestion
In the [code/solutions](code/solutions), you can find the completed solution for each lab. This can be used as a reference,
in case you get stuck or things don't work as intended.
```

## Local development

> To set up local development environment, please follow the instructions in
> [local development](docs/LOCAL_DEVELOPMENT.md) file.

Reviewer 2: ```suggestion
To set-up a local development environment for changing the workshop, please follow the instructions in
[local development](docs/LOCAL_DEVELOPMENT.md) file.
```

Clarify that this is for people wanting to change the workshop, not intended for people following it.
